src/platform/library/include/platform/lib/memory.h

```diff
@@ -66,10 +65,12 @@ static inline uint32_t arch_get_stack_size(void)
 
 #define SRAM_BANK_SIZE	0x10000
 #define LP_SRAM_SIZE SRAM_BANK_SIZE
-#define HP_SRAM_SIZE SRAM_BANK_SIZE
+#define HP_SRAM_SIZE (SRAM_BANK_SIZE * 47)
```

Comment:
47 has no special meaning, right? It's just to match one of existing platforms? You could use 4 or 32 or 1024 here or whatever? Maybe at least add a comment there

Reply:
yes, let me add a comment for it.